rest/app.py

```diff
@@ -93,3 +107,7 @@ def set_task_action():
 @app.errorhandler(404)
 def not_found(error):
     return render_template('error.html'), 404
+
+
+if __name__ == '__main__':
+    app.run(host='0.0.0.0', port=5000, debug=True)

```

Comment:
Add _ToDo_ with moving port to Config after 
https://github.com/stasya72008/highway-to-hell/projects/2#card-11155636

Reply:
Please,  check this issue - https://github.com/stasya72008/highway-to-hell/projects/2#card-11096922
Maybe Solution with adding _main_ will resolve this issue

Reply:
Proposed solution does not work, unfortunately